packages/template-retail-react-app/app/pages/checkout/confirmation.jsx

```diff
@@ -102,6 +108,28 @@ const CheckoutConfirmation = () => {
     const CardIcon = getCreditCardIcon(order.paymentInstruments[0].paymentCard?.cardType)
 
     const submitForm = async (data) => {
+        const saveShippingAddress = async (customerId) => {
+            try {
+                const shippingAddress = order.shipments[0].shippingAddress
+                let {id, ...shippingAddressWithoutId} = shippingAddress
+                const bodyShippingAddress = {
+                    addressId: nanoid(),
```

Comment:
Hey @dannyphan2000 .. what was the reasoning to have to pluck the id out of the order shipping address and replace it with a new id supplied via nanoId? Could we not use a static value here like 'default'.